packages/plugin-solana/src/actions/transfer.ts

```diff
@@ -207,21 +207,43 @@ export default {
             const transaction = new VersionedTransaction(messageV0);
             transaction.sign([senderKeypair]);
 
-            // Send transaction
-            const signature = await connection.sendTransaction(transaction);
-
-            console.log("Transfer successful:", signature);
-
-            if (callback) {
-                callback({
-                    text: `Successfully transferred ${content.amount} tokens to ${content.recipient}\nTransaction: ${signature}`,
-                    content: {
-                        success: true,
-                        signature,
-                        amount: content.amount,
-                        recipient: content.recipient,
-                    },
+            // Send transaction with client retry
+            const timeoutMs = 60000;
+            const startTime = Date.now();
+            while (Date.now() - startTime < timeoutMs) {
+                const transactionStartTime = Date.now();
+                const signature = await connection.sendTransaction(transaction, {
+                    maxRetries: 0,
+                    skipPreflight: true,
```

Comment:
why are we skipping these?

Reply:
I optimized this implementation for transaction landing.

Many RPC-providers have their nodes behind loadbalancers. It might happen that when you fetch the latest blockhash (line 202), you get it from a node that is ahead of the one your transaction is being simulated on, resulting in a "blockhash not found" error. This is quite common.

https://www.helius.dev/blog/how-to-land-transactions-on-solana#how-do-i-land-transactions

Reply:
Let me know if you anything else from me